theme/templates/generic/includes/comment.html

```diff
@@ -16,6 +17,13 @@ <h4>{{ comment.user|contact }}
                 <a href="#reply-{{ comment.id }}" class="reply">
                     <span class="glyphicon glyphicon-share-alt"></span> {% trans "Reply" %}
                 </a>
+                {%  if user.is_authenticated %}
+                    {% if comment.user == user or user.is_staff or self_access_level == 'owner' %}
```

Comment:
I see is_staff is used in the template but is_superuser is used in the view function. Should they be made consistent?

Reply:
well, probably.  As far as I can tell, they resolve the same way, it's just is_staff seems to be the mezzanine term for is_superuser?  This template was already using is_staff, so I went with it here.  I'll try updating the endpoint to check for is_staff.